app/views/categories/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="container mt-5 pt-5">
+  <div class="row">
+    <% @categories.each do |category| %>
+      <div class="col-6"> <!-- Bootstrap class for 2 cards per row on medium screens and up -->
+        <%= link_to category_user_transactions_path(category), class: 'category-link link' do %>
+          <div class="card mb-4">
```

Comment:
- [ ] Please make sure that the `add new category` and `add new transaction` buttons styles match with the given design as requested in the project requirements to follow the design guidelines.

### Your design

![image](https://github.com/NoelLincoln/budget-app/assets/83803414/812e2f82-3f06-4a5d-8bff-f1f3d40ce584)

![image](https://github.com/NoelLincoln/budget-app/assets/83803414/e0e9d540-2970-406a-acbf-0dccd02da383)

### Given design

![image](https://github.com/NoelLincoln/budget-app/assets/83803414/7b90a6ba-3600-4482-86ab-ee3bda2cfe9a)